examples/src/main/java/com/example/CommonsTextFuzzer.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Code Intelligence GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.example;
+
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import org.apache.commons.text.StringSubstitutor;
+import com.code_intelligence.jazzer.api.Jazzer;
```

Comment:
`format.sh` seems to be quite non-portable to me, even the one on main (I think it is written to work on Ubuntu maybe)

anyways, I will rather try to address the issues found by the CI job manually to avoid changing other files (although I no longer can see any jobs triggered for this PR, for some reason, they seem to be stuck)

Reply:
i did run the clang formatter on the new Java files, not sure if it fixes it on CI because workflows are blocked on approval - please reopen if still problematic